src/include/platform/ConfigurationManager.h

```diff
@@ -79,6 +79,13 @@ class ConfigurationManager
     CHIP_ERROR GetDeviceCertificate(uint8_t * buf, size_t bufSize, size_t & certLen);
     CHIP_ERROR GetDeviceIntermediateCACerts(uint8_t * buf, size_t bufSize, size_t & certsLen);
     CHIP_ERROR GetDevicePrivateKey(uint8_t * buf, size_t bufSize, size_t & keyLen);
+    CHIP_ERROR GetDeviceDACCertificate(uint8_t * buf, size_t bufSize, size_t & dacLen);
+    CHIP_ERROR GetDevicePAICertificate(uint8_t * buf, size_t bufSize, size_t & paiLen);
+    CHIP_ERROR GetDevicePAACertificate(const uint8_t * skid, size_t skidLen, uint8_t * buf, size_t bufSize, size_t & paaLen);
```

Comment:
The (input only, right?) skid should _definitely_ be a `ByteSpan` here.  And this needs documentation explaining what that argument does.